storage/innobase/xtrabackup/src/xbcloud/xbcloud.cc

```diff
@@ -405,6 +431,14 @@ static my_bool get_one_option(int optid,
         extra_http_headers.insert(parse_http_header(argument));
       }
       break;
+    case OPT_CURL_RETRIABLE_ERRORS:
+      if (argument != nullptr) {
+        CURLcode code = static_cast<CURLcode>(atoi(argument));
+        if (code < CURLcode::CURL_LAST) {
+          curl_retriable_errors.push_back(code);
```

Comment:
I feel this way of passing option is not ideal. Some command-line tools (in fact mysqld too) pick the last one as the choice. if you pass buf_pool_size twice, it will take the last one.

We want the user to pass a comma-separated list.  the semantics are clear and straightforward. Pass the option only once.